app.test.js

```diff
@@ -0,0 +1,26 @@
+import { addList, displayList } from './src/modules/app.js';
+
+describe('Add and Remove Testing', () => {
+  document.body.innerHTML = `
+  <input id='toDoInput'>
+  <button id='addBtn'></button>
+  <div id='toDoListItemContainer'></div>
+  `;
+
+  test('adding item to list', () => {
+    addList('paint', false, 0);
+    const localGet = JSON.parse(localStorage.getItem('listStorage'));
+    expect(localGet.length).toBe(1);
+  });
+
+  test('Removing item from list', () => {
+    addList('work', false, 1);
+    const localGet = JSON.parse(localStorage.getItem('listStorage'));
+    expect(localGet.length).toBe(2);
+    localGet.pop();
+    localStorage.setItem('listStorage', JSON.stringify(localGet));
+    displayList();
+    const todolist = document.getElementById('toDoListItemContainer');
+    expect(todolist.childElementCount).toBe(1);
+  });
+});
```

Comment:
- [ ] I am not able to run tests please fix task
![test](https://user-images.githubusercontent.com/44745855/195387621-87f4e009-d1ec-4c0d-a4d0-513d55589034.png)

Reply:
![Screenshot (21)](https://user-images.githubusercontent.com/65858136/195389283-bd3a1162-b9fc-460f-ab68-6c9d62fa372e.png)

it does run on my end as well as my partners